src/spira_training/shared/core/models/wav.py

```diff
@@ -0,0 +1,5 @@
+from typing import NewType
+
+import torch
+
+Wav = NewType("Wav", torch.Tensor)
```

Comment:
isso aqui eu queria evitar se possivel.... dentro do nosso core reusando um tipo to pytorch. 
eu coloquei o pytorch sempre como adapter la no model training